scripts/check-update.sh

```diff
@@ -11,20 +11,20 @@ else
     echo "No .envrc"
 fi
 
-CHANNEL=$1
-echo "CHANNEL: $CHANNEL"
+RELEASE_CHANNEL=$1
+echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-if [ "$CHANNEL" = "dev" ]; then
-    CURRENT_VERSION=$(cat scripts/prisma_version_unstable)
+if [ "$RELEASE_CHANNEL" = "dev" ]; then
+    CURRENT_VERSION=$(cat scripts/prisma_version_insider)
```

Comment:
The file `scripts/prisma_version_insider` would need a final bump, so, it contains the last Prisma CLI version published when this is merged.